src/config.ts

```diff
@@ -205,6 +205,14 @@ const logLevel = envOrString(process.env.LOG_LEVEL, 'info');
 
 const notifications = {
   desktop: process.env.DESKTOP_NOTIFICATIONS === 'true',
+  apns: {
+    apnsAuthKey: envOrString(process.env.APNSAUTHKEY),
+    apnsKeyId: envOrString(process.env.APNSKEYID),
+    apnsTeamId: envOrString(process.env.APNSTEAMID),
+    apnsProduction: envOrBoolean(process.env.APNSPRODUCTION),
+    apnsDeviceToken: envOrString(process.env.APNSDEVICETOKEN),
+    apnsBundleId: envOrString(process.env.APNSBUNDLEID)
```

Comment:
```suggestion
    apnsAuthKey: envOrString(process.env.APNS_AUTHKEY),
    apnsKeyId: envOrString(process.env.APNS_KEYID),
    apnsTeamId: envOrString(process.env.APNS_TEAMID),
    apnsProduction: envOrBoolean(process.env.APNS_PRODUCTION),
    apnsDeviceToken: envOrString(process.env.APNS_DEVICETOKEN),
    apnsBundleId: envOrString(process.env.APNS_BUNDLEID)
```

Do you mind using the `APNS_` prefix?

Reply:
Done